README.md

```diff
@@ -31,6 +51,18 @@ This stylesheet is transforming an XML document by removing any "sec" element wi
 
 TODO: We need an expression of whether we believe this is work for biorXiv, encoda or EPP team.
 
+### [/src/2022.07.26.501569/move-ecole-into-institution.xsl](/src/2022.07.26.501569/move-ecole-into-institution.xsl)
+
+Adjusts 2 of the affiliations where the department is being treated as an address rather than in the institution. This would have to require a bioRxiv adjustment. There is no way we could discern from the XML otherwise.
+
+There may be another example in 10.1101/2022.10.21.513138:
+
+```
+<aff id="a1"><label>1</label><institution>Univ-Bordeaux, Centre de Recherche Cardio-thoracique de Bordeaux</institution>, U1045, D&#x00E9;partement de Pharmacologie, CIC1401, Pessac, <country>France</country></aff>
+```
+
+It is clear that we could not hope to automatically capture these without improving the XML representation.
+
```

Comment:
This is partially incorrect. It can be fixed by treating affiliations as mixed content (pulling in the text content of aff as well as `institution`, `country` etc.), and the change has been implemented in encoda v0.121.1 - see https://github.com/elifesciences/enhanced-preprints-issues/issues/343.

Reply:
Right, so we can get at the value by other means. Will update the documentation.

Reply:
Thanks

Reply:
I've amended this documentation

Reply:
Thanks LGTM